package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "videojs-contrib-quality-levels",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
Usually this is done outside of PRs, to allow us to package multiple changes together (and prevent conflicts before PRs get merged). Would it be possible to set this back to 2.0.3?